### PR TITLE
Add getCustomName (with a breaking change)

### PR DESCRIPTION
**Pull request by u9g**
the breaking change is that you now have to call prismarine-entity with a minecraft version. This is so that we can call prismarine-chat. In reality, I guess we could just specify any version but it's best to just give a minecraft version to prismarine-entity. I also made this an actual class to not use deprecated functions (`util.inherits`)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "prismarine-entity",
-  "version": "1.2.0",
+  "version": "2.0.0",
```
Comment (rom1504) [inline package.json:3]: This should be another commit

Comment (u9g) [reply package.json]: done

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
   "devDependencies": {
     "@types/node": "^17.0.4",
     "prismarine-item": "^1.4.0",
-    "standard": "^16.0.1"
+    "standard": "^16.0.1",
+    "prismarine-chat": "^1.4.1"
```
Comment (rom1504) [inline package.json:32]: Should be a dep, not a dev dep

Comment (u9g) [reply package.json]: done
